core/templates/dev/head/css/oppia.css

```diff
@@ -386,6 +377,16 @@ p:last-child {
   font-size: 1em !important;
 }
 
+.oppia-rte ol, ul {
```

Comment:
This means that this will be applied to `ol` elements inside elements with class `oppi-rte` and to **all** `ul` elements. You probably want `.oppia-rte ol, .oppia-rte ul`, you can also merge this with the next rule, because they're the same.